src/database/all.sql

```diff
@@ -274,3 +284,17 @@ INSERT INTO bid_status (request_status_id, status_name) values (4, 'Cancelled');
 
 COMMIT;  
   
+------------------------------ 
+-- From: 340_insert_bid_status.sql 
```

Comment:
Hey Carl,
Please use these status' instead, this is what Front-End has worked off of. I think at this point it will be easier if we just conform on this.

New
Pending Acceptance
Pending Completion
Complete
Archived